space/spacecore/storage/sqlitestorage/sql.go

```diff
@@ -0,0 +1,119 @@
+package sqlitestorage
+
+const sqlCreateTables = `
+CREATE TABLE IF NOT EXISTS spaces  (
+  id text not null primary key,
+  header text not null,
+  settingsId text not null,
+  aclId text not null,
+  hash text not null default '',
+  oldHash text not null default '',
+  isCreated boolean not null default false,
```

Comment:
Could you please explain why we need this field? I don't understand, because from my point of view, if space exist in this table and field IsDeleted=false, then it's considered as existing 😅

Reply:
I don't know, I've implemented the existing interface :) As @mcrakhman  said, it's hard to explain :)